scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/extension/ClassDefinitionSetWithExtensionMethods.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.extension
+
+import pl.touk.nussknacker.engine.ModelData
+import pl.touk.nussknacker.engine.definition.clazz.{ClassDefinition, ClassDefinitionSet}
+
+final case class ClassDefinitionSetWithExtensionMethods private (value: ClassDefinitionSet)
+
+object ClassDefinitionSetWithExtensionMethods {
```

Comment:
This class became small and readable. Nice! I'm thinking about one more little step that we can do to make it even better. We could rename it to sth like `ExtensionMethodsDefinitionsEnricher` and rename `apply` to `enrichWithExtensionMethods`. It could be even in the same file as `ExtensionMethods`. Thanks to that we won't introduce another entity, and keeps all extension methods registry logic together. It will be easier to find out where to add another extension method. WDYT?

Reply:
Done